### DISCUSSION TITLE
How-To Register as AD App?

### DISCUSSION THREAD

**Discussion by jazucco-oeis** [Power-Maverick/DataverseDevTools-VSCode#98] [Q&A] [answered]
I've been trying to add Dataverse DevTools as an Azure AD App so that I can use in Visual Studio Code to work with my Power Apps environments. I've tried following the instructions at https://learn.microsoft.com/en-us/power-apps/developer/data-platform/walkthrough-register-app-azure-active-directory and while I've been able to add a connection the connection does not seem to do anything. 

Are there any more detailed instructions for setting up this tool as AD App? Key items I think I'm missing: 

1. Redirect URI
2. Client Secret, when I create a secret for the AD App do I use the "Value" or the "Secret ID" when creating a connection?

Any additional help would be greatly appreciated.

**Comment by Power-Maverick** [comment, marked as answer]
When you use Microsoft Login Prompt; you will be automatically redirected to a browser to authenticate. If you do not have admin access; just copy the generated URL and send it to the admin. After admin receives the URL sent by you they can authenticate and approve the app on the Enterprise level enabling its usage across all organization. 
P.S. admin needs to be a global admin in Azure
